src/index.js

```diff
@@ -157,8 +157,9 @@ class HawkWebpackPlugin {
 
         body.append('file', file, {
           filename: map.name
-        })
-        body.append('release', releaseId)
+        });
+        body.append('release', releaseId);
+        body.append('commits', []);
```

Comment:
do we need sending empty commits array?

Reply:
It's required field, collector won't accept release without this field

Reply:
Will we store such an empty array in a DB?